demo/async-demo.py

```diff
@@ -14,11 +14,21 @@
 
 print("--- urllib3 using Trio ---")
 import trio
-trio.run(main, "trio")
+trio.run(main)
 
 print("\n--- urllib3 using AnyIO ---")
 import anyio
-anyio.run(main, "anyio")
+anyio.run(main)
```

Comment:
What is the intention here? Would it not make more sense to do `anyio.run(main, backend='trio')` since now you're running anyio using the asyncio backend twice.

Reply:
It just tests that the default AnyIO backend works but that's not too useful indeed. I don't think we want to test AnyIO support for Trio, so maybe I should just leave this out?

Reply:
Probably, yeah.